SubmittedAssignments/Sanjeevanee_balure_03_/index.html

```
@@ -0,0 +1,809 @@
+        <div class="profile">
+           <img src="https://www.bing.com/th/id/OIP.MLiBwIjHUfAjRrA4mq0SDgHaHa?w=197&h=211&c=8&rs=1&qlt=90&o=6&dpr=1.3&pid=3.1&rm=2" alt="Weddings & Celebrations" />
+        </div>
+        <h3>Weddings & Celebrations</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut elit tellus, luctus nec ullamcorper mattis.</p>
+        <span>Up to 150 guests</span>
+      </div>
+
+      <!-- Card 2 -->
+      <div class="card">
+        <div class="profile">
+         <img src="https://th.bing.com/th/id/OIP.xi8n2dtN-FQxVBXV1MoTZAHaEZ?w=324&h=192&c=8&rs=1&qlt=90&o=6&dpr=1.3&pid=3.1&rm=2"alt="Corporate Events" /> 
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)